apis/v1alpha1/defaulting.go

```diff
@@ -427,3 +433,58 @@ func defaultReadinessProbe() *corev1.Probe {
 		FailureThreshold: 10,
 	}
 }
+
+// Same as the default rolling update strategy of Deployment.
+func defaultRollingUpdateForDeployment() *appsv1.RollingUpdateDeployment {
```

Comment:
We should not set the default RollingUpdate value here? kubernetes will set the default value for deployment and statefulset.

Reply:
The default value is the same as the native K8s configuration.